### ISSUE TITLE
The Description field in debian/control has whitespace-only lines

### ISSUE THREAD

**Issue by hxw** [Debian/dh-make-golang#30]
Looks like trailing whitespace is treated as text rather than blank lines.  The build process then fails
because it treats these lines as blank.

The regexp for parsing description needs to be equivalent to: `s/^[[:space:]]*$/ ./` when parsing the description field.

I am using: dh-make-golang    0.0~git20150803.0.bd09cd9-1     amd64 in Ubuntu 15.10

(Thanks for an excellent program)

**Comment by stapelberg** [comment]
Can you provide an example Go package with which you observe this? That’d make it much easier to test and understand.

**Comment by hxw** [comment]
Here is the snippet of a failing package (cihub/seelog):

~~~
000008b0  6c 6c 6f 77 69 6e 67 20  63 6f 6d 6d 61 6e 64 3a  |llowing command:|
000008c0  0a 20 2e 0a 20 0a 20 67  6f 20 67 65 74 20 2d 75  |. .. . go get -u|
000008d0  20 67 69 74 68 75 62 2e  63 6f 6d 2f 63 69 68 75  | github.com/cihu|
000008e0  62 2f 73 65 65 6c 6f 67  0a 20 2e 0a 20 0a 20 49  |b/seelog. .. . I|
000008f0  4d 50 4f 52 54 41 4e 54  3a 20 49 66 20 79 6f 75  |MPORTANT: If you|
00000900  20 61 72 65 20 6e 6f 74  20 75 73 69 6e 67 20 74  | are not using t|
~~~
~~~
 document and follow the instructions. Once you're ready, execute the
 following command:
 .
 
 go get -u github.com/cihub/seelog
 .
         
 IMPORTANT: If you are not using the latest release version of Go, check
 out this wiki page (https://github.com/cihub/seelog/wiki/Notes-on-)
~~~

It actually seems to be happening before/after the "sample code" triple backticks.

**Comment by stapelberg** [comment]
This issue was already fixed with commit 155e78c88255afe0a2ad7403171e1ec66d497132 in issue #11.

**Comment by hxw** [comment]
Excellent, Thanks
